mirdata/datasets/beatles.py

```diff
@@ -286,3 +285,36 @@ def _fix_newpoint(beat_positions):
     beat_positions[beat_positions == "0"] = "4"
 
     return beat_positions
+
+
+@core.docstring_inherit(core.Dataset)
+class Dataset(core.Dataset):
+    """The beatles dataset
+    """
+
+    def __init__(self, data_home=None):
+        super().__init__(
+            data_home,
+            index=DATA.index,
+            name="beatles",
+            track_object=Track,
+            bibtex=BIBTEX,
+            remotes=REMOTES,
+            download_info=DOWNLOAD_INFO,
+        )
+
+    @core.copy_docs(load_audio)
```

Comment:
This is a bit obscure IMO, either we go with something more explicit (if possible)  or will have to add a clear explanation in the docs :/

Reply:
I agree it's obscure... I couldn't find a better way to include the load functions along with the docs. The only more explicit way to do it is to copy paste the docs from each loader function :(. I'll try to make a clear explanation in the contributing

Reply:
Yeah, I couldn't think of an alternative now either. I'm good with keeping it like this and adding comments on `CONTRIBUTING.md`. Should we add comments on @cached_property as well?